install/install.py

```diff
@@ -157,19 +159,20 @@ def install(self, cmd_arguments):
 
                 if fixsetuptools == 'y':
                     # backup the original symlink
-                    print("Backing up the original symlink...")
+                    Colorizer.normal("Backing up the original symlink...")
                     ts = time.time()
                     human_timestamp = datetime.fromtimestamp(ts).strftime('%Y-%m-%d-%H:%M:%S')
 
                     # backup the original symlink
                     self.run_command("mv /usr/lib/python2.7/dist-packages/setuptools.egg-info /usr/lib/python2.7/dist-packages/setuptools.egg-info" + "-BACKUP-" + human_timestamp)
 
-                    print("The original symlink exists at /usr/lib/python2.7/dist-packages/setuptools.egg-info-BACKUP-" + human_timestamp)
+                    Colorizer.info("The original symlink exists at /usr/lib/python2.7/dist-packages/setuptools.egg-info-BACKUP-" +
+                                   human_timestamp)
```

Comment:
If it has be over one line, better use an extra indent like below:

```python
Colorizer.info(
    'string bablabla' % param)  # if string + param fits in one line
Colorizer.info(
    'string bablabla' %
    param)  # otherwise
```

In what you did, it becomes very annoying to keep the alignment, especially when it is not a multiple of 4. It looks stupid but hitting spacebar 3 times is much more annoying after a while vs. hitting tab once.